ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml

```diff
@@ -0,0 +1,131 @@
+import QtQuick 2.12
+import QtGraphicalEffects 1.13
+
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import utils 1.0
+import "../controls"
+import "../stores"
+
+OnboardingBasePage {
+    id: root
+
+    property string mnemonicText
+    signal exit()
```

Comment:
Suggestion: `OnboardingBasePage` has the `finished()` signal that maybe could be used instead of `exit()` one. Indeed it is for the same purpose!